src/pymovements/utils/checks.py

```diff
@@ -111,7 +114,9 @@ def check_two_kwargs(**kwargs) -> None:
         raise ValueError('there must be exactly two keyword arguments in kwargs')
 
 
-def check_is_mutual_exclusive(**kwargs) -> None:
+def check_is_mutual_exclusive(
```

Comment:
I would use `Any` here, as the type of the kwarg really does not matter

Reply:
done